helix-term/src/keymap.rs

```diff
@@ -62,247 +154,300 @@ impl DerefMut for Keymaps {
-            key!('K') => Command::keep_selections,
+            "K" => keep_selections,
             // TODO: and another method for inverse
 
             // TODO: clashes with space mode
-            key!(' ') => Command::keep_primary_selection,
+            "space" => keep_primary_selection,
 
-            // key!('q') => Command::record_macro,
-            // key!('Q') => Command::replay_macro,
+            // "q" => record_macro,
+            // "Q" => replay_macro,
 
-            // ~ / apostrophe => change case
             // & align selections
             // _ trim selections
 
             // C / altC = copy (repeat) selections on prev/next lines
 
-            key!(Esc) => Command::normal_mode,
-            key!(PageUp) => Command::page_up,
-            key!(PageDown) => Command::page_down,
-            ctrl!('b') => Command::page_up,
-            ctrl!('f') => Command::page_down,
-            ctrl!('u') => Command::half_page_up,
-            ctrl!('d') => Command::half_page_down,
-
-            ctrl!('w') => Command::window_mode,
+            "esc" => normal_mode,
+            "pageup" => page_up,
+            "pagedown" => page_down,
+            "C-b" => page_up,
+            "C-f" => page_down,
+            "C-u" => half_page_up,
+            "C-d" => half_page_down,
+
+            "C-w" => {
+                "w" => rotate_view,
+                "C-w" => rotate_view,
+                "h" => hsplit,
+                "v" => vsplit,
+                "q" => wclose
+            },
 
             // move under <space>c
-            ctrl!('c') => Command::toggle_comments,
-            key!('K') => Command::hover,
+            "C-c" => toggle_comments,
+            "K" => hover,
 
             // z family for save/restore/combine from/to sels from register
 
-            // supposedly ctrl!('i') but did not work
-            key!(Tab) => Command::jump_forward,
-            ctrl!('o') => Command::jump_backward,
-            // ctrl!('s') => Command::save_selection,
-
-            key!(' ') => Command::space_mode,
-            key!('z') => Command::view_mode,
+            // supposedly "C-i" but did not work
+            "tab" => jump_forward,
+            "C-o" => jump_backward,
+            // "C-s" => save_selection,
+
+            "space" => {
+                "f" => file_picker,
+                "b" => buffer_picker,
+                "s" => symbol_picker,
+                "w" => {
+                    "w" => rotate_view,
+                    "C-w" => rotate_view,
+                    "h" => hsplit,
+                    "v" => vsplit,
+                    "q" => wclose
+                },
+                "y" => yank_joined_to_clipboard,
+                "Y" => yank_main_selection_to_clipboard,
+                "p" => paste_clipboard_after,
+                "P" => paste_clipboard_before,
+                "R" => replace_selections_with_clipboard,
+                "space" => keep_primary_selection
+            },
```

Comment:
Err, doesn't this means the infobox and generated doc will be gone?

Reply:
Added infobox back in in d4ed5e31f201c4c17d54c0353b0bd9cf236182a7, headings need to be worked in, will add it soon.